### ISSUE TITLE
training parameters

### ISSUE THREAD

**Issue by maliho0803** [natanielruiz/deep-head-pose#64]
what is the best training parameters?

**Comment by natanielruiz** [comment]
Not really. I used the exact code that is online to achieve the results in the paper. Try to monitor your validation loss and do early stopping when it is minimal.

**Comment by maliho0803** [comment]
thank you for your reply!!! I use batch_size of 64, maybe this will decress the accuracy?

One more question, I validate AFLW2000 during training every epoch, but it get different results
as I test AFLW2000 just using testing code with the same model, I check the weight is the same and I have set model.eval(), do you know why I get such difference?

**Comment by tx1994108** [comment]
@maliho0803 .hello, I want to know what the parameters of your training are.  I still have 10 losses. And the loss is still very volatile. thank you.

**Comment by shayanjoya** [comment]
Hi Nathaniel,

I am aware this is an old paper, but can you please advise if I am missing something.

I have trained hopenet using the same hyperparameters that you mentioned: batch_size 128, lr 1e-5 on 300W_LP, and am testing on AFLW2000 on the filtered images.

My best results is Yaw: 9.6283, Pitch: 6.6494, Roll: 5.4697, MAE: 7.25

Evaluated on 25th epoch:
Test error in degrees of the model on the 1969 test images. Yaw: 12.6357, Pitch: 7.2710, Roll: 6.3822
Evaluated on 20th epoch:
Test error in degrees of the model on the 1969 test images. Yaw: 11.8453, Pitch: 6.7829, Roll: 6.0533
Evaluated on 15th epoch:
Test error in degrees of the model on the 1969 test images. Yaw: 12.3951, Pitch: 6.8435, Roll: 5.9623
Evaluated on 10th epoch:
Test error in degrees of the model on the 1969 test images. Yaw: 12.2416, Pitch: 6.8511, Roll: 5.8133
Evaluated on 5th epoch:
Test error in degrees of the model on the 1969 test images. Yaw: 9.6283, Pitch: 6.6494, Roll: 5.4697   [BEST] MAE: 7.25
Evaluated on 4th epoch:
Test error in degrees of the model on the 1969 test images. Yaw: 10.1037, Pitch: 7.0748, Roll: 5.4427
Evaluated on 3rd epoch
Test error in degrees of the model on the 1969 test images. Yaw: 9.2807, Pitch: 8.6684, Roll: 5.9980
Evaluated on 2nd epoch
Test error in degrees of the model on the 1969 test images. Yaw: 9.9768, Pitch: 7.3000, Roll: 5.8216
Evaluated on 1st epoch
Test error in degrees of the model on the 1969 test images. Yaw: 8.4784, Pitch: 7.0443, Roll: 5.9930
